blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -106,42 +110,61 @@ function parseArticleItem(item, index, arcSite, phrases, id, customFields) {
         figureImageClassName += allowedFloatValue === 'left' ? ' article-body-image-container--mobile-left-float' : ' article-body-image-container--mobile-right-float';
       }
 
-      return (url && url.length > 0) ? (
-        <figure
-          className={figureImageClassName}
-          key={key}
-        >
-          <Image
-            resizedImageOptions={resizedImageOptions}
-            url={url}
-            alt={altText}
-            smallWidth={widthsObject.small}
-            smallHeight={0}
-            mediumWidth={widthsObject.medium}
-            mediumHeight={0}
-            largeWidth={widthsObject.large}
-            largeHeight={0}
-            breakpoints={getProperties(arcSite)?.breakpoints}
-            resizerURL={getProperties(arcSite)?.resizerURL}
-          />
-          <figcaption>
-            <ImageMetadata
-              subtitle={subtitle}
-              caption={caption}
-              credits={credits}
-              vanityCredits={vanityCredits}
+      if (url) {
+        const ArticleBodyImage = () => (
+          <figure
+            className={figureImageClassName}
+            key={key}
+          >
+            <Image
+              resizedImageOptions={resizedImageOptions}
+              url={url}
+              alt={altText}
+              smallWidth={widthsObject.small}
+              smallHeight={0}
+              mediumWidth={widthsObject.medium}
+              mediumHeight={0}
+              largeWidth={widthsObject.large}
+              largeHeight={0}
+              breakpoints={getProperties(arcSite)?.breakpoints}
+              resizerURL={getProperties(arcSite)?.resizerURL}
             />
-          </figcaption>
-        </figure>
-      ) : null;
+            <figcaption>
+              <ImageMetadata
+                subtitle={subtitle}
+                caption={caption}
+                credits={credits}
+                vanityCredits={vanityCredits}
+              />
+            </figcaption>
+          </figure>
+        );
+
+        // if link url then make entire image clickable
+        if (link) {
+          return (
+            <a
+              href={link}
+              aria-hidden="true"
```

Comment:
using aria-hidden for best practices around links for accessibility

Reply:
Why would this link need to be removed from user using assistive technologies?

Reply:
ah I was going off of some of the other tickets. but that we didn't want duplication for assistive tech; maybe I misunderstood how it was done in other places